contrib/create-user.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+function waitForAuthOperatorProgressing {
```

Comment:
To allow that this script is called from different places, we need to get the root or base folder (cloned console folder).

You can test this by running the `create-user.sh` script from different folders. For example

from console folder:
```
contrib/create-user.sh
```

or from the integration-test folder:

```
cd frontend/packages/dev-console/integration-tests
../../../../contrib/create-user.sh
```

```suggestion
set -exuo pipefail

BASEDIR=$(dirname "$0")/..

function waitForAuthOperatorProgressing {
```